xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/XWikiSkinsSheet.xml

```diff
@@ -149,6 +149,7 @@
     (((
       {{html}}&lt;button class="btn btn-default btn-xs"&gt;$services.icon.renderHTML('cross')&lt;/button&gt;{{/html}}
     )))
+    )))
```

Comment:
I cannot see any code that adds the corresponding `(((`, was this missing before?

Reply:
Sorry, it was a mistake, I don't know what I did here...
Addressed in https://github.com/xwiki/xwiki-platform/pull/2642/commits/18cc897f5255e678a50fb4f70fa31e914338ed7d :+1: